lib/ssh_helper/ssh.go

```diff
@@ -0,0 +1,157 @@
+package ssh_helper
```

Comment:
__[golint]__
* [golint] don't use an underscore in package name

<!-- Comment by [SideCI](https://sideci.com) -->